umbraco-cloud/set-up/project-settings/secrets-management.md

```diff
@@ -68,6 +68,16 @@ You can use the method, getting it from the System namespace in .NET as below:
 
 `_secretMessage = Environment.GetEnvironmentVariable("SecretMessage");`
 
+Secrets can also be used to override AppSettings defined in appsettings.json files.
+
+In order for this to work, when adding the secret, the Key value should be all the settings' names joined by double underscores. 
+
+For example, to change the Serilog's default options under "Serilog" > "MinimumLevel" > "Default" key, the Secret key would look like this:
```

Comment:
Is this a reference a setting in appsettings.json?

```suggestion
For example, to change the Serilog's default options under `Serilog:MinimumLevel:Default`, the Secret key would look like this:
```

Reply:
Yes, that is right